byzcoin/bcadmin/main.go

```diff
@@ -703,6 +695,10 @@ func init() {
 			Value:  getDataPath(cliApp.Name),
 			Usage:  "path to configuration-directory",
 		},
+		cli.BoolFlag{
+			Name:  "export, x",
+			Usage: "redirects the transaction to stdout",
+		},
```

Comment:
is it not possible to move the export flag under contracts? is it because you're using `c.GlobalBool("export")`?

I prefer to minimise the use of globals in functions, it's perfectly fine if `AddTransactionAndWaitWithOption` has an extra argument.

Reply:
Using --export at the contract level was indeed the plan, but I am not able to access flags declared before the final subcommand.

Reply:
can you try using the parent context? https://godoc.org/github.com/urfave/cli#Context.Parent